ObjC/FXPageControl.h

```diff
@@ -1,7 +1,7 @@
 //
 //  FXPageControl.h
 //
-//  Version 1.4
+//  Version 1.5
```

Comment:
Moving this file out of the Sources folder will make it not be included in the library when installed through Cocoapods and it is also not marked as dependency. Please either revert or add FXPageControl as dependency in the podspec.

Reply:
That's a big mistake from my side! I ran a `pod spec lint XLPagerTabStrip.podspec` instead of `pod lib lint XLPagerTabStrip.podspec`. Fix is on the way